x-pack/legacy/plugins/reporting/server/routes/lib/get_document_payload.ts

```diff
@@ -97,6 +96,4 @@ function getDocumentPayloadFn(server: any) {
     // send a 503 indicating that the report isn't completed yet
     return getIncomplete(status);
   };
-}
-
-export const getDocumentPayloadFactory = oncePerServer(getDocumentPayloadFn);
+};
```

Comment:
Sort-of related: https://github.com/elastic/kibana/pull/49250/files#diff-a21790aa8cc3b71aadd0e5c6630fbed0 I am going through and replacing the `any` types with real types. 

I'll also add a (Jest) unit test for this file in my PR

Reply:
Yeah, I wanted to do some work here, but since we're looking at the `exportTypesRegistry` for refactor, I just left it